src/components/album/AlbumName.jsx

```diff
@@ -0,0 +1,18 @@
+import "./AlbumName.css"
+import PropTypes from 'prop-types'
```

Comment:
Be consistent throughout your whole project with whether you're using single or double quotes.